client/gutenberg/extensions/prev-next/index.js

```diff
@@ -0,0 +1,75 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import wp from 'wp';
+const { __ } = wp.i18n;
```

Comment:
Care to put those `const` under a "Module variables" section like we do throughout Calypso?

Reply:
it's not really a module variable. this code should disappear soon so I don't care what we do with it. it will be displayed by the `@wordpress` packages we're currently importing into Calypso